README.md

```diff
@@ -241,11 +289,13 @@ sudo setcap cap_net_bind_service=+ep leapchat
 
 ## Documentation Links
 
-Currently, some of the packages we use are slightly out of date. Here are some quick documentation links that might prove useful:
-
-[React Bootstrap version 0.33.1 docs](https://react-bootstrap-v3.netlify.app/components/tooltips/)
+Open via `npm`:
 
-Search for available [Font Awesome Icons](https://fontawesome.com/v5/search)
+```
+$ npm docs bootstrap
+$ npm docs react-bootstrap
+$ npm docs react-icons
```

Comment:
This makes a good deal more sense than keeping the links themselves up-to-date.

Reply:
Ha, sounds good